freud/order.pyx

```diff
@@ -377,13 +396,25 @@ cdef class Hexatic(_PairCompute):
 cdef class Translational(_PairCompute):
     R"""Compute the translational order parameter for each particle.
 
+    The translational order parameter is used to measure order in the bonds
+    of 2D systems. The translational order parameter for a particle :math:`i`
+    and its :math:`n` neighbors :math:`j` is given by a sum over the
+    neighbors, treating the 2D vectors between each pair of particles as a
+    complex number with real part corresponding to the x-component of the
+    vector and imaginary part corresponding to the y-component of the vector,
+    divided by a normalization constant :math:`k`:
+
+    :math:`\psi\left( i \right) = \frac{1}{k} \sum_j^n x_{ij} + y_{ij} i`
+
+    The translational order parameter as written above is **complex-valued**.
+
     .. note::
         **2D:** :class:`freud.order.Translational` is only defined for 2D
         systems. The points must be passed in as :code:`[x, y, 0]`.
 
     Args:
         k (float, optional):
-            Symmetry of order parameter. (Default value = :code:`6.0`).
+            Normalization of order parameter (Default value = :code:`6.0`).
```

Comment:
This parameter is still related the the symmetry of the translational order right? Ideally the description in the docs should say something about both.

Reply:
Since this value is explicitly used as a normalization (the final result is divided by `k`), and it's not necessarily indicative of any kind of `k`-fold symmetry in the particle's neighbors, I think this change clarifies how the value is used. Also we don't really know how the Translational order parameter has been used in practice before and we're probably going to deprecate and remove it in freud 3.0, as I discussed with @vyasr in #655.

Reply:
Since this PR contains a bunch of changes to Translational that are unrelated to the original goal of the PR, we should just apply the deprecation here as well.

Reply:
@vyasr I already created an issue #659 and will make sure that gets done before the next release -- but I would like to get this PR merged as-is, rather than stall it further. The existing changes in this PR just aimed to improve its docs, because of the specific change request in #655. 😕